sources/cassandra/terraform.tfvars

```diff
@@ -0,0 +1,17 @@
+common_params = {
+  spanner_project_id   = "test-project"
+  spanner_instance_id  = "test-instance"
+  spanner_database_id  = "test-db"
+  region               = "us-central1"
+  zone                 = "us-central1-a"
+  instance_count       = "10"
+  instance_name_prefix = "node-10-zdm"
+  network              = "default"
+  machine_type         = "n2-standard-8"
+  service_account_key  = "path/to/local/key/file"
```

Comment:
Sorry I might be missing the entire context, but why do we need `service_account_key`?
Can we avoid it?

Many times production projects (including ones in `google.com`) have service account key creation disabled
This is as per Google's recommendations for Customer's security posture
https://cloud.google.com/iam/docs/best-practices-for-managing-service-account-keys

So it's quite possible that we might encouter a Cx who can not generate a key from an SA.

Reply:
The service account key is to allow the sidecar allow access to Spanner. This seems consistent with the official documentation of how the cassandra-to-spanner-proxy recommends running it.

Reply:
Added a todo to find more secure ways of doing this